src/sentry/web/frontend/organization_auth_settings.py

```diff
@@ -111,6 +115,9 @@ def handle_existing_provider(
         provider = auth_provider.get_provider()
 
         if request.method == "POST":
+            if provider.is_partner:
+                return HttpResponse("Can't disable partner authentication provider", status=405)
+
```

Comment:
Is it only a POST that can update the settings? Just checking to make sure there's no PUT too

Reply:
Right now there's no PUT implementation.